### PR TITLE
feat: hook-up push notifcation in aave mod as testing ground

### PR DESCRIPTION
**Pull request by petrovska-petro**
plays with #15

### CODE REVIEW

File: contracts/modules/RevokeModule.sol [hunk 1]
```diff
@@ -3,6 +3,7 @@ pragma solidity ^0.8.4;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "interfaces/ISafe.sol";
+import "interfaces/push/INotification.sol";
```
Comment (gosuto-inzasheru) [inline contracts/modules/RevokeModule.sol:6]: maybe move this to `BaseModule` as well?

**Review by gosuto-inzasheru — COMMENTED** [review]
tests are all successful

notis worked on relayer ui

think the natspec tags are a bit all over the place, but can solve in #17 

just maybe move push protocol code to base module?

File: contracts/modules/RevokeModule.sol (hunk 1 above)
Comment (gosuto-inzasheru) [reply contracts/modules/RevokeModule.sol:6]: actually might as well move ierc and isafe there as well, no?

Comment (petrovska-petro) [reply contracts/modules/RevokeModule.sol:6]: interesting while trying to settled them back to base module, brownie complains on compiling 😢

Comment (gosuto-inzasheru) [reply contracts/modules/RevokeModule.sol:6]: lets not do this for now

**Review by gosuto-inzasheru — APPROVED** [review]

**Pull request merged**